pkg/ddl/executor.go

```diff
@@ -565,20 +565,21 @@ func (e *executor) ModifySchemaSetTiFlashReplica(sctx sessionctx.Context, stmt *
 		}
 
 		job := &model.Job{
```

Comment:
maybe in future, we can have a `newJob` function to avoid the job constructor changes need to be edit at many places

Reply:
Good idea～